### ISSUE TITLE
Would you benefit from CMSIS-Driver?

### ISSUE THREAD

**Issue by ReinhardKeil** [sensorplatforms/open-sensor-platform#9]
We did review the open-senson-platform and the project looks interesting.

Would it be beneficial to use CMSIS-Driver interfaces in this project?
If so we could help you to drive it forward.

Thanks
Reinhard

**Comment by riansanderson** [comment]
Yes, your help would be much appreciated.  We have pull requests for CMSIS-RTOS and would like to use the CMSIS peripheral drivers for our hardware abstraciton.  We have some suggested extensions to the CMSIS peripheral driver interfaces.  Why don't we setup a call for next week.  You can email me at rsa......on@audience.com
